canvas_modules/harness/cypress/e2e/canvas/nodes-resize.cy.js

```diff
@@ -34,8 +34,14 @@ describe("Test adding resizing nodes", function() {
 	it("Test all attached and detached links appear OK after moving nodes around", function() {
 		cy.verifyNumberOfPortDataLinks(8);
 
+		// TODO -- Fix when autoselect is available.
```

Comment:
There are 20 `TODO` comments in this PR. Remove TODO comments for all tests which are working.

Reply:
Hi Neha I am keeping these TODO comments in the PR as we discussed.

Reply:
@nmgokhale This is the issue described here https://github.ibm.com/NGP-TWC/wdp-abstract-canvas/issues/3760
which we ought to fix if we can.  If we can fix it the tests would return to doing the automatic selection on drag.